schema/spacewalk/upgrade/susemanager-schema-4.4.8-to-susemanager-schema-4.4.9/010-encrypt_credentials-task.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO rhnTaskQueue (id, org_id, task_name, task_data)
+SELECT sequence_nextval('rhn_task_queue_id_seq'), id, 'upgrade_satellite_encrypt_credentials', 0
```

Comment:
I was unable to find this task. Can you point me in the right direction MC?

Reply:
This is a migration which is handled during tomcat startup. Check
`java/code/src/com/redhat/rhn/manager/satellite/UpgradeCommand.java`

Reply:
thanks for the clarification